index.js

```diff
@@ -30,6 +30,9 @@ class CoverageBookend extends BookendInterface {
             return;
         }
 
+        // Set 'enabled' config into specified plugin instance.
+        config[pluginName].enabled = config.enabled;
```

Comment:
Is this object indent correct? Is `config.enabled` right?
https://github.com/screwdriver-cd/screwdriver/pull/1758/files

Reply:
Config is set here.
https://github.com/screwdriver-cd/screwdriver/blob/master/bin/server#L77

config object will be like this;
```
{ enabled: true,
  plugin: 'sonar',
  sonar:
   { sdApiUrl: 'http://172.21.202.48:9001',
     sonarHost: 'https://sonar-cd.sandbox.screwdriver.corp.yahoo.co.jp',
     adminToken: '5f9b2722f3d26e866150cde1bb64622d919977cf',
     sdUiUrl: 'http://172.21.202.48:9000',
     enabled: true } }
```

Reply:
thanks, it's my misunderstanding.